projects/addon-mobile/components/sheet/components/sheets-host/sheets-host.template.html

```diff
@@ -16,8 +16,5 @@
             (close)="close(item)"
         ></tui-sheet>
     </div>
-    <div
-        class="t-overlay"
-        (click)="close(items[0])"
-    ></div>
+    <div class="t-overlay"></div>
```

Comment:
It has `pointer-events: auto` if it follows a closeable sheet (with class `._overlay` or `.t-wrapper_overlay`):
![image](https://user-images.githubusercontent.com/11832552/179004187-a5eb3663-37b1-48cc-aefd-ebc0281b9be6.png)